applications/luci-app-tailscale-community/root/usr/share/rpcd/ucode/tailscale.uc

```diff
@@ -0,0 +1,266 @@
+#!/usr/bin/env ucode
+
+'use strict';
+
+import { access, popen, readfile, writefile, unlink } from 'fs';
+import { cursor } from 'uci';
+
+const uci = cursor();
+// Keep in doubt about its usefulness
+const env_script_path = "/etc/profile.d/tailscale-env.sh";
+const ori_env_script_content = `#!/bin/sh
+# This script is managed by luci-app-tailscale-community.
+uci_get_state() { uci get tailscale.settings."$1" 2>/dev/null; }
+if [ "$(uci_get_state daemon_reduce_memory)" = "1" ]; then export GOGC=10; fi
+TS_MTU=$(uci_get_state daemon_mtu)
+if [ -n "$TS_MTU" ]; then export TS_DEBUG_MTU="$TS_MTU"; fi
+`;
+const env_script_content = replace(ori_env_script_content, /\r/g, '');
```

Comment:
It seems like this should go into a separate file, either in this package, or in the packages repo. Look at e.g. luci-app-statistics